benchmark/super_resolution_model_zoo/LICENSE

```diff
@@ -0,0 +1,5 @@
+# Licenses
```

Comment:
Since we put tfjs in our benchmark, we need to specify tfjs's license here to state that we are using some other project. I think i included a README in the same folder.